app/models/sal3_batch_requests_batch.rb

```diff
@@ -41,9 +41,9 @@ class Sal3BatchRequestsBatch < ActiveRecord::Base
   def self.locations
     {
       'Data Control' => 'DC',
-      'Special Collections - Redwood City' => 'RW',
+      'Redwood City (Special Collections materials)' => 'RW',
       'Archive of Recorded Sound' => 'DA',
-      'Archive of Recorded Sound - Redwood City' => 'RA',
+      'Redwood City (Archive of Recorded Sounds materials)' => 'RA',
```

Comment:
Sorry, I should have asked for "Sound" instead of "Sounds"

```suggestion
      'Redwood City (Archive of Recorded Sound materials)' => 'RA',
```